test-fixtures/urn/f11/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "description": "We shouldn't lock the open URN for legal deposit use",
```

Comment:
I don't understand this description

Reply:
Me neither. I just transferred it from old style test.